mlflow/gateway/app.py

```diff
@@ -110,7 +114,7 @@ async def index():
     async def health():
         return {"status": "OK"}
 
-    @app.get("/gateway/routes/{route_name}")
+    @app.get("/api/2.0/gateway/routes/{route_name}")
```

Comment:
Should we reference `MLFLOW_GATEWAY_ROUTE_PREFIX` here instead?

Reply:
good point. I think I briefly tried interpolation (which isn't allowed at all) but string concat works just fine :) updated!